src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    static String getRandomBall() {
+        Random random = new Random();
+        int randomNumber = random.nextInt(101);
+
+        Ball ball = new Ball();
+        ball.setNumber(randomNumber);
+        ball.setColour(getRandomColor());
+
+        return String.valueOf(ball);
```

Comment:
You can directly return the ball object instead of converting it to a string. If you need a string representation, override the toString method in the Ball class.